shell-and-git/branches-pull-requests.md

```diff
@@ -0,0 +1,14 @@
+# Git Branches and Pull Requests
+
+## Important Terminology
+
+- **Pull request:** essentially a request to add all the changes from a branch into the main branch.
```

Comment:
The "essentially" does not add extra information to the sentence, so I would recommend to remove it.